docker/test.sh

```diff
@@ -0,0 +1,19 @@
+# - VERY rudimentary test script to run latest + specific branch image builds and test them all by running `--version`
+# TODO: create a real test suite
+set -e
+
+echo "\n\n"
```

Comment:
Doesn't this need `-e` to be effective? I'd just do `printf`s for control characters.

Reply:
🔪